debian/debhelper-build-stamp

```diff
@@ -0,0 +1 @@
+wb-mqtt-confed
```

Comment:
> In compat 10 (or later), dh creates a stamp file debian/debhelper-build-stamp after the build step(s) are complete to avoid re-running them.

`debian/debhelper-build-stamp` в gitignore надо.

Reply:
поправлено